mesa/space.py

```diff
@@ -71,11 +71,11 @@ def __init__(self, height, width, torus):
 
         self.grid = []
 
-        for y in range(self.height):
-            row = []
-            for x in range(self.width):
-                row.append(self.default_val())
-            self.grid.append(row)
+        for x in range(self.width):
+            col = []
```

Comment:
Look at test_grid.py in the tests directory for understanding the structure of representation. Also see the note which should be visible in the PR conversation history as well. That comment with one example could be added into the documentation part of this file for understanding of future developers.